exercises/practice/reverse-string/reverse-string.rakutest

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.dirname;
+use ReverseString;
+
+cmp-ok( # begin: c3b7d806-dced-49ee-8543-933fd1719b1c
+    switch-round(""),
+    "eq",
+    "",
+    "an empty string",
+) &&
```

Comment:
I recommend not using `&&` to join tests together. If the first test fails, the second will not run, changing the number of expected tests for the test runner.

I would suggest wrapping these as separate statements within a [subtest](https://docs.raku.org/type/Test#sub_subtest).